docs/zz_generated.seed.yaml

```diff
@@ -123,6 +123,8 @@ spec:
           availability_zone: ""
           # Used for automatic network creation
           dns_servers: []
+          # Optional: List of enabled flavors for the given datacenter
+          enabled_flavors: null
```

Comment:
Please adjust https://github.com/Happy2C0de/kubermatic/blob/add-support-to-restrict-openstack-flavors/cmd/example-yaml-generator/main.go#L136-L142 to show an empty array (`[]`) as the example default, so it's obvious that this is an array and not a comma-separated string or whatever.